### PR TITLE
Rs 2023.01.19

### PR DESCRIPTION
**Pull request by rschlaikjer**
Quickie
2023-01-19-224133_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 13/13_01_01_CIEL13_5C.txt [hunk 1]
```diff
@@ -146,7 +167,8 @@ She won't wake up ever again. After all, her mind is completely dead!
 [sha:f8e9c7373b5d6dc654de0714bcb0f2ab7f301f3c]{
 -- Page 670, Offset 35194.
 -- 見せつけられた罪に耐えきれず、自分から覚醒を拒否した大罪人。卑怯で下劣な肉人形なんだから！」
-She's a grievous sinner who refused to wake up because she couldn't bear the weight of her own crimes. In the end, she's nothing more than a cowardly and vile human puppet!"
+She's such a profligate sinner that she can't bear the weight of her own sins, and won't even let herself wake up!
+ In the end, she's nothing more than a cowardly and vile meat puppet!"
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_01_CIEL13_5C.txt:171]: Sin and sinner felt a little too repetitive for me. Maybe it's fine?
肉人形 as "meat puppet" just sounds a bit too bizarre, and doesn't quite get across what she means.
なんだから！should probably have more impact.
>She's a profligate sinner who was unable to bear witness to her own crimes who refuses to even let herself wake up.
>A cowardly, despicable, lifelike little doll, and nothing more!

File: script/Ciel/Day 13/13_01_01_CIEL13_5C.txt [hunk 2]
```diff
@@ -146,7 +167,8 @@ She won't wake up ever again. After all, her mind is completely dead!
...
 }
 [sha:96457ae90661b461bb3b612270a37f08f01fc2d3]{
 -- Page 671, Offset 35195.
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_01_CIEL13_5C.txt:174]: Ellipsis check

File: script/Ciel/Day 13/13_01_01_CIEL13_5C.txt [hunk 3]
```diff
@@ -176,7 +198,8 @@ Because every time I call out to her, I can see her body react slightly.
 [sha:2941aac4de2d071068556fa2fbfff84c08dfce8f]{
 -- Page 671, Offset 35200.
 -- その名前を聴く度に、びくりと、泣くように肩が震えている。
-Whenever she hears her name, her shoulders jump up and tremble as though she's about to burst into tears.
+Whenever she hears her own name,
+ her shoulders tremble as though she's about to burst into tears.
 }
 [sha:42cb57ddad86a05a52752e216140a8f9c8a84941]{
 -- Page 672, Offset 35201. Mods: @k, @e.
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_01_CIEL13_5C.txt:205]: Kinda like having stubborn in both...
>You stubborn little thing, you.

>Oh, but maybe it's that same stubbornness which helped...

File: script/Ciel/Day 13/13_01_01_CIEL13_5C.txt [hunk 4]
```diff
@@ -186,38 +209,51 @@ Whenever she hears her name, her shoulders jump up and tremble as though she's a
...
+ Ah, but maybe it's that same stubbornness of yours that helped you survive my Mystic Eyes?
 }
 [sha:519ff7d9610ddb7e0d43bd32e1bc88ea1881a35e]{
 -- Page 672, Offset 35203.
 -- でも無駄な努力よ。どうあってもアナタは助からない。命乞いをきく耳もないし、逃がすのなんて絶対にない。
-But your efforts are wasted. No matter what happens, your fate is sealed. I won't let you off the hook even if you beg for mercy, and there's absolutely no chance you're getting away.
+It still won't save you, though.
+ No matter what happens, your fate is sealed.
+ I won't let you off the hook even if you beg for mercy,
+ and you don't stand a snowball's chance in hell of escaping.
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_01_CIEL13_5C.txt:220]: Last two sentences are a bit lengthy and clumsy.
>Beg for mercy all you want, I'm %{i}never%{/i} letting you out of here alive.

File: script/Ciel/Day 13/13_01_01_CIEL13_5C.txt [hunk 5]
```diff
@@ -186,38 +209,51 @@ Whenever she hears her name, her shoulders jump up and tremble as though she's a
...
 -- アナタはここで、夢の中のどんな死に方より痛くて酷くて可笑しい死に方で殺されるの」
-You shall die right here, and I'm going to make it more excruciating, ruthless, and amusing than anything you saw in the dream."
+You're going to die right here,
+ and it's going to be more excruciating, brutal, and freakish than anything you saw in that dream."
 }
 [sha:774df2bb727afb9b7cb9a725f93aadcbf4286f78]{
 -- Page 673, Offset 35205.
 -- 「あ。でもその前に、せっかくだから聞いておこっと。
-"Ah. But before that, there's something I want to ask you.
+"Ah. But before we begin, there's something I want to ask you.
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_01_CIEL13_5C.txt:231]: I'd probably prefer a comma.
Also, voice.
>Ah, but before we begin, I may as well ask.

File: script/Ciel/Day 13/13_01_01_CIEL13_5C.txt [hunk 6]
```diff
@@ -186,38 +209,51 @@ Whenever she hears her name, her shoulders jump up and tremble as though she's a
...
 }
 [sha:4a779d45089e18efddaedc1f16bede9a620ed32e]{
 -- Page 673, Offset 35206.
 -- これって<希|レ><少|ア>かつ一度きりの美味しい機会だし」
-I mean, this is a once-in-a-lifetime rare and delectable opportunity, after all."
+I mean, this is the sort of delectable opportunity you only get once in a lifetime, after all."
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_01_CIEL13_5C.txt:236]: A bit too long.
>I mean, this kind of delectable opportunity comes only once a lifetime.

File: script/Ciel/Day 13/13_01_01_CIEL13_5C.txt [hunk 7]
```diff
@@ -186,38 +209,51 @@ Whenever she hears her name, her shoulders jump up and tremble as though she's a
...
 -- Page 674, Offset 35207.
 -- 「ねえ、わたしの<夢|なか>はどうだった？　濃密すぎて興奮した？　可愛すぎて欲情した？　<中|ク><毒|セ>になるぐらい酩酊した？
 // ????????????????????????????????????
-"Tell me, how did it feel to be inside of me? Was it so dense that it aroused you? Was it so adorable that it turned you on? Was it so intoxicating that it made you an addict?
+// Arach has rubbed off on her too much
+// Reading 濃密 as rich (taste, contents), potent (scent), like it's so overwhelming / luxurious with sensation?
+"Tell me, how did it feel to be inside of me?
+ Did you get off on how intense it was?
+ Did you get all riled up by how cute it was?
+ Did you get hooked on how intoxicating it was?
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_01_CIEL13_5C.txt:247]: The dream part should probably still be there, but with u
欲情 should be way more explicit, honestly. "Riled up" is a little too mild.
I also think you can sort of change all the "it" to "I"

>Tell me, how did it feel to dream inside of me?
>Did you get off on how intense it was?
>Did you get aroused by how charming it was?
>Did you get hooked on how intoxicating it was?

File: script/Ciel/Day 13/13_01_01_CIEL13_5C.txt [hunk 8]
```diff
@@ -186,38 +209,51 @@ Whenever she hears her name, her shoulders jump up and tremble as though she's a
...
 }
 [sha:cadc7244bb26014f51ab33caaf471443b1420989]{
 -- Page 674, Offset 35208.
 -- っていうか、そいつがどのくらい最悪でわたしがどのくらい正しいか分かってくれた？　心の底から謝って、二人仲良くわたしに殺されたくなったかしら……！」
- More importantly, did you realize how deplorable she truly is? Did you realize that I've been in the right all this time? Perhaps you've finally decided to beg for my forgiveness and submit to death...!"
+More importantly,
+ did you realize how deplorable she truly is?
+ Did you realize that I've been in the right all this time?
+ Did you finally decide it's time to beg for forgiveness,
+ and let me kill the two of you right here and now?"
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_01_CIEL13_5C.txt:256]: Bit simpler. Also, the 二人仲良くわたしに殺されたくなったかしら part has lost its impact.
>Or like, did you realize how awful she truly is, and how in the right I am?
>Did you perhaps get in the mood to apologize to me, and let me kill the both of you right there!

File: script/Ciel/Day 13/13_01_01_CIEL13_5C.txt (hunk 7 above)
Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_01_CIEL13_5C.txt:247]: not sure what you mean by "but with u" here

Comment (Petrikow) [reply script/Ciel/Day 13/13_01_01_CIEL13_5C.txt:247]: I don't either. Don't worry about it.

**Pull request merged**